docs/how-to/create-a-router/configure-ensembler.md

```diff
@@ -25,6 +29,19 @@ In addition, a fallback route may be configured whose results will be used at ru
 It is not possible to select as the fallback response a route that has traffic rules associated to it.
 {% endhint %}
 
+### Custom Experiment Engines
+For routers configured with Custom Experiment Engines, the router will return a response from one of the routes 
+corresponding to the route name that is found within the treatment configuration returned by the experiment 
+engine. At run time, the router will attempt to access the route name within the treatment configuration received via a 
+user-configured path. 
+
+A fallback route also has to be configured in order to capture cases whereby the route name found in the treatment 
+configuration does not correspond to any of the routes configured, or when the user-configured path is invalid with 
+respect to the treatment configuration received.
+
+The UI for Standard Ensemblers with Custom Experiment Engines is consumed directly as a remote component that is 
```

Comment:
This line is better removed from the user docs - is it more of a dev doc detail.